packages/paypal-js/types/v6/components/find-eligible-methods.d.ts

```diff
@@ -36,8 +36,71 @@ export type FindEligibleMethodsGetDetails<T extends FundingSource> =
           ? PayLaterEligiblePaymentMethodDetails
           : BaseEligiblePaymentMethodDetails;
 
+/**
+ * Result object returned by `findEligibleMethods()` containing methods to check payment method eligibility and retrieve payment method details.
+ *
+ * @remarks
+ * This interface is returned by the SDK's `findEligibleMethods()` method. It provides
+ * utilities to determine which PayPal payment methods are available and to retrieve details for payment setup.
+ *
+ * @example
+ * ```typescript
+ * const paymentMethods = await sdkInstance.findEligibleMethods({
+ *   currencyCode: "USD",
+ * });
+ *
+ * if (paymentMethods.isEligible("paypal")) {
+ *   const paypalPaymentSession = sdkInstance.createPayPalOneTimePaymentSession({
+ *     onApprove: (data) => console.log('Payment approved:', data),
+ *   });
+ *   // set up button
+ *   ...
```

Comment:
How does it render these `...`s, since they aren't valid syntax? Should we comment these out?
```suggestion
 *   // ...
```

Reply:
Meh, I just commented them out. 😁